rust/operator-binary/src/security/oidc.rs

```diff
@@ -92,3 +104,116 @@ pub(crate) async fn check_or_generate_oidc_admin_password(
 pub fn build_oidc_admin_password_secret_name(nifi: &NifiCluster) -> String {
     format!("{}-oidc-admin-password", nifi.name_any())
 }
+
+pub fn add_oidc_config(
+    provider: &oidc::AuthenticationProvider,
+    oidc: &ClientAuthenticationOptions,
```

Comment:
suggestion: Make it clearer what type this parameter holds.

```suggestion
    client_auth_options: &ClientAuthenticationOptions,
```

_Note: Needs adjustments below._